### PR TITLE
Add missing null check for envFile

### PR DESCRIPTION
**Pull request by tajila**
Add missing null check for envFile

Fixes https://github.com/eclipse-openj9/openj9/issues/13829

Signed-off-by: Tobi Ajila <atobia@ca.ibm.com>

### CODE REVIEW

File: jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java [hunk 1]
```diff
@@ -371,77 +371,79 @@ public CRIUSupport registerRestoreEnvFile(Path envFile) {
...
-					}
-
-					String file = envFile.toAbsolutePath().toString();
-
-					try (BufferedReader envFileReader = new BufferedReader(new FileReader(file))) {
-						/* [IF JAVA_SPEC_VERSION < 17] */
-						SecurityManager manager = System.getSecurityManager();
-						if (manager != null) {
-							manager.checkRead(file);
+		if (this.envFile != null) {
```
Comment (DanHeidinga) [inline jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java:374]: Personally, I'd be tempted to do:
```java
if (this.envFile == null) {
    return;
}
```
and leave the rest of the code at the previous indent level.  We don't enforce single-return point in java code.

**Review by DanHeidinga — APPROVED** [review]

**Comment by DanHeidinga** [comment]
Jenkins test sanity xlinux jdk11

**Comment by DanHeidinga** [comment]
@tajila we don't have a special criu pr build/test setup yet, right?

**Comment by tajila** [comment]
> @tajila we don't have a special criu pr build/test setup yet, right?

No, we havent sorted out the machine situation yet

**Pull request merged**